probe-rs-tools/src/bin/probe-rs/main.rs

```diff
@@ -40,6 +43,17 @@ struct Cli {
     /// Enable logging to the default folder. This option is ignored if `--log-file` is specified.
     #[clap(long, global = true, help_heading = "LOG CONFIGURATION")]
     log_to_folder: bool,
+    #[clap(
+        long,
+        short,
+        global = true,
+        help_heading = "LOG CONFIGURATION",
+        value_name = "PATH",
+        require_equals = true,
+        num_args = 0..=1,
```

Comment:
Why do we need these?

Reply:
That's so we can use a flag with a default value :) It's required to make `default_missing_value` work. It's explained in the clap docs.